tests/dashboard/network-graphs.test.js

```diff
@@ -26,5 +27,4 @@ describe('Network Graphs', () => {
       .then(title => expect(title).to.equal(test.expected));
     });
   });
-
```

Comment:
Should include:
- Are all network interfaces present in the drop down?
- Do they all have a graph
- Change of interface in dropdown generates a new graph
- Change of resolution tests 
- Validate that data streams in (e.g. should wait a for the graph to update)
- Validate empty graphs - need to think how this is possible